cluster-provision/centos9/scripts/vm.sh

```diff
@@ -151,5 +157,10 @@ exec qemu-system-x86_64 -enable-kvm -drive format=qcow2,file=${next},if=virtio,c
   -vnc :${n} -cpu host,migratable=no,+invtsc -m ${MEMORY} -smp ${CPU} \
   -serial pty -M q35,accel=kvm,kernel_irqchip=split \
   -device intel-iommu,intremap=on,caching-mode=on -device intel-hda -device hda-duplex -device AC97 \
+  -device qemu-xhci,id=bus1 -device qemu-xhci,id=bus2  \
```

Comment:
Do we want to have it always enabled? Otherwise, we could add an option as its the case for SCSI and NVMe emulation.

Reply:
We can do it on demand I don't have a problem with it.